doc/source/release/1.15.0-notes.rst

```diff
@@ -234,25 +239,27 @@ and support several Array API compatible array libraries in addition to NumPy
   GPUs.
 - `scipy.special.ndtr` is now more efficient.
 
-`scipy.stats` improvements
-==========================
+``scipy.stats`` improvements
+============================
 - A new probability distribution infrastructure has been added for the
-  implementation of univariate, continuous distributions with speed,
-  accuracy, and memory advantages:
+  implementation of univariate, continuous distributions. It has several
+  speed, accuracy, memory, and interface advantages compared to the
+  previous infrastructure. See :ref:`rv_infrastructure` for a tutorial.
 
-  - `scipy.stats.Normal` represents the normal distribution with the new
-    interface. In typical cases, its methods are faster and more accurate than
-    those of `scipy.stats.norm`.
   - Use `scipy.stats.make_distribution` to treat an existing continuous
     distribution (e.g. `scipy.stats.norm`) with the new infrastructure.
     This can improve the speed and accuracy of existing distributions,
-    especially for methods not overridden with custom formulas in the
-    implementation.
-
-- `scipy.stats.Mixture` has been added to represent mixture distributions.
-- Instances of `scipy.stats.Normal` and the classes returned by
-  `scipy.stats.make_distribution` are supported by several new mathematical
-  transformations.
+    especially those with methods not overridden with distribution-specific
+    formulas.
+  - `scipy.stats.Normal` and `scipy.stats.Uniform` are pre-defined classes
+    to represents the normal and uniform distributions, respectively.
```

Comment:
`represents` -> `represent` I think?

Apart from that, the diff and rendered version look good to me, so I think this can be squash-merged to facilitate backporting.

Reply:
```suggestion
    to represent the normal and uniform distributions, respectively.
```